internal/watchdog/watchdog_test.go

```
@@ -81,6 +83,54 @@ var _ = Describe("Watchdog", func() {
 	})
 })
 
+var _ = Describe("Watchdog start signalling", func() {
+
+	var ctx context.Context
+
+	BeforeEach(func() {
+		var cancel context.CancelFunc
+		ctx, cancel = context.WithCancel(context.Background())
```

golangci-lint (2.12.2)

[error] nested context in function literal

(fatcontext)